### ISSUE TITLE
Reference subworkflow error: Permission denied

### ISSUE THREAD

**Issue by lkhilton** [LCR-BCCRC/lcr-modules#46]
I tried pointing my Snakefile to the pre-built reference directory at `/projects/bgrande/reference_files`, but I get the following error on a dry run: 
```
PermissionError: [Errno 13] Permission denied: '/projects/bgrande/reference_files/.snakemake/log/2020-06-10T143949.122053.snakemake.log'
```

The Snakefile I'm running is at `/home/lhilton/rrDLBCL_trios`, and looks like this: 
```
import oncopipe as op 

# Load default module config 
configfile: "/home/lhilton/repos/lcr-modules/modules/manta/2.0/config/default.yaml"

# Load project config
configfile: "config/config.yaml"

# Load samples
SAMPLES = op.load_samples("data/metadata/samples.tsv")
config["lcr-modules"]["_shared"]["samples"] = SAMPLES 

subworkflow reference_files: 
    workdir: 
        "/projects/bgrande/reference_files"
    snakefile: 
        "/home/lhilton/repos/lcr-modules/workflows/reference_files/1.0/reference_files.smk"
    configfile: 
       "/home/lhilton/repos/lcr-modules/workflows/reference_files/1.0/config/default.yaml"


# Include manta module Snakefile
include: "/home/lhilton/repos/lcr-modules/modules/manta/2.0/manta.smk"
```

**Comment by lkhilton** [comment]
I was about to implement this very thing before the file system crash. I will test it out now that I can get this working on gphost02.

**Comment by BrunoGrandePhD** [comment]
The workaround is fine for now. The long-term fix will be tracked in #37.
